engine/src/cmd/unit_const_cache.h

```diff
@@ -3,35 +3,38 @@
 #include "hashtable.h"
 #include <string>
 #include <gnuhash.h>
+#include <functional>
 
 class Mesh;
-class ConstHasher;
 
-class StringIntKey
+struct StringIntKey
```

Comment:
Per standard, struct is just a class with public by default. However, I've found that by convention (and compat with C) it's better to keep struct to data only stuff, and class for data+function stuff. $0.02